src/NUnitFramework/framework/Internal/Tests/ParameterizedFixtureSuite.cs

```diff
@@ -54,5 +64,14 @@ public override string TestType
                     : "ParameterizedFixture";
             }
         }
+
+        /// <summary>
+        /// Overriden to return a Parameterized Fixture Suite
+        /// </summary>
+        /// <param name="filter">Filter to apply</param>
```

Comment:
`Overriden` is a typo, but I'd also recommend copying the entire base method's documentation exactly.